ansible/configs/selinux-policy/post_software.yml

```diff
@@ -14,14 +14,12 @@
         - "Here are your credentials to access the lab environment:"
         - ""
         - "Password: {{ student_password | d(hostvars[groups.bastions.0].student_password)}}"
-        - "IP Address: {{ hostvars[groups.bastions.0].public_ip_address }}"
+        - "SSH Connection: {{ student_user }}@{{ hostvars[groups.bastions.0].public_ip_address }}"
         - ""
-        - "To access your lab environment please follow the guide:"
+        - "To access the lab instructions, check the following guide:"
         - ""
         - "https://2020-summit-labs.gitlab.io/selinux-policy-bookbag/"
         - ""
-        - "Whenever you see instructions on the guide like &lt;PASSWORD&gt; or &lt;IP_ADDRESS&gt; you replace with credentials provided here."
-        - ""
 
     - name: print out user.data
       agnosticd_user_info:
```

Comment:
I don't understand why you have this section for `agnosticd_user_info/data` and then you have another section at line 41. Can we merge those and avoid duplication or is there a reason they are separate?